packages/react-instantsearch/src/connectors/connectHierarchicalMenu.test.js

```diff
@@ -25,11 +25,13 @@ describe('connectHierarchicalMenu', () => {
       };
 
       results.getFacetValues.mockImplementationOnce(() => ({}));
+      results.getFacetValues.mockImplementation(() => ({}));
```

Comment:
why do you need `results.getFacetValues.mockImplementation(() => ({}));` on top of `results.getFacetValues.mockImplementationOnce(() => ({}));` ?